ee-feature-pack/common/src/main/resources/modules/system/layers/base/org/wildfly/extension/opentelemetry-api/main/module.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2021, Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags. See the copyright.txt file in the
+  ~ distribution for a full listing of individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+
+<module xmlns="urn:jboss:module:1.9" name="org.wildfly.extension.opentelemetry-api">
```

Comment:
I am not expert in wildfly building. But does it mean module `org.wildfly.extension.opentelemetry-api` will be included in EAP Base, when this module is located in `wildfly-ee-feature-pack-common`?

Reply:
@jasondlee My understanding is this feature should end up in Microprofile part of server - EAP XP.  Can you confirm it will not end up in EAP Base part?

Reply:
Thanks for raising it again @mchoma , I was preparing the conversions and faced the same. We could deal with them at conversion level, but this should be treated upstream instead, keeping all modules, licenses, and resources used by this extension at the galleon-pack level instead of the ee- variant, ideally handled as [wildfly-elytron-oidc-client](https://github.com/wildfly/wildfly/blob/795c1fc286afd32c1a474f6aeb430e5889eb3869/elytron-oidc-client/galleon-common/pom.xml#L40) does. Let's wait for Jason's feedback.

Reply:
I can see modules in ee-dist, what I believe is analogy of EAP Base
```
pwd
/wildfly/ee-dist/target/wildfly-25.0.0.Final-SNAPSHOT/modules/system/layers/base/org/wildfly/extension
[mchoma@localhost extension (master)]$ tree -L 1
.
...
├── opentelemetry
├── opentelemetry-api
...
```

Reply:
It is my understanding that this will ship with XP4 (that port is underway at the moment) but that it will also ship with EAP 8 (though I'm not a PM so don't quote me on that :).